packages/inter-protocol/src/vaultFactory/vaultHolder.js

```diff
@@ -33,19 +26,20 @@ const HolderI = M.interface('holder', {
   makeCloseInvitation: M.call().returns(M.promise()),
   makeTransferInvitation: M.call().returns(M.promise()),
 });
+
+/** @type {{ [name: string]: [ description: string, valueShape: Pattern ] }} */
+const PUBLIC_TOPICS = {
+  vault: ['Vault holder status', M.any()],
```

Comment:
Is the second element of this array intentionally unused? I was trying to suggest a better name for PUBLIC_TOPICS, but can't actually determine what that would be.

Reply:
Oh yeah I dropped the runtime use of `valueShape` in some iteration. It had been used in a `mustMatch` before recording. I'll restore it.

I also intend for the shape to specify the TS type so that all the methods that take or return `value` are typed. Maybe a `TypedPattern` for now that takes a generic for what the type is. I don't think I can genericize Pattern because it's `any`.

Reply:
the `valueGuard` is in there now. I left them as `M.any()` because I'm not confident in how they'll be updated through upgrade, but I think this is the right API for specifying the value type

Reply:
I guess I would suggest switching from tuple to record:
```suggestion
/** @type {{ [name: string]: { description: string, valueShape: Pattern } }} */
const PUBLIC_TOPICS = {
  vault: { description: 'Vault holder status', valueShape: M.any() },
```